metadata.yaml

```diff
@@ -26,4 +30,3 @@ storage:
     type: filesystem
     location: /data
     minimum-size: 10G
-min-juju-version: 2.8.6
```

Comment:
if memory serves me well, this should probably be changed to `min-juju-version: 2.9.0` instead of being removed

Reply:
Better still would be to drop `min-juju-version` and use the new `assumes` metadata entry to require a given juju version :)